ci/download-core-sdk.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+# Modify the PINNED_CORE_SDK_VERSION variable in `ci/includes/pinned-tools.sh` and run this script to download
+# the new version of the CoreSDK and supporting tools.
+
+set -e -u -x -o pipefail
+
+cd "$(dirname "$0")/../"
+
+source ci/includes/pinned-tools.sh
+source ci/includes/profiling.sh
+
+function unpackTo() {
+  local SOURCE=$1
+  local TARGET=$2
+
+  mkdir -p "${TARGET}"
+  unzip -o -q "${SOURCE}" -d "${TARGET}"
+}
+
+PINNED_CORE_SDK_VERSION="$(cat core-sdk.version)"
+
+CORE_SDK_DIR="$(pwd)/build/core_sdk"
+UNITY_PROJECT_DIR="$(pwd)/workers/unity"
+NATIVE_DEPENDENCIES_PATH="${UNITY_PROJECT_DIR}/Assets/Plugins/Improbable/Core"
+MANAGED_DEPENDENCIES_PATH="${UNITY_PROJECT_DIR}/Assets/Plugins/Improbable/Sdk"
```

Comment:
Do  we want to clean these directories before unpacking the `dll` and `so` files.

Reply:
Done